sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/converters/SearchResultHelper.java

```diff
@@ -47,4 +50,12 @@ static void setHighlights(SearchResult searchResult, Map<String, List<String>> h
     static void setJsonSerializer(SearchResult searchResult, JsonSerializer jsonSerializer) {
         accessor.setJsonSerializer(searchResult, jsonSerializer);
     }
+
+    static void setRerankerScore(SearchResult searchResult, Double rerankerScore) {
+        accessor.setRerankerScore(searchResult, rerankerScore);
+    }
+
+    static void setCaptions(SearchResult searchResult, List<CaptionResult> captions) {
```

Comment:
Question on the API: Does the order matters for the List<CaptionResult>?

Reply:
Order doesn't matter